rakefile

```diff
@@ -7,19 +7,26 @@ include AbsHelper
 require './tests/helpers/perf_results_helper'
 include PerfResultsHelper
 
+REF_ARCH_STD, REF_ARCH_LARGE = ["S","L"]
+
 task :default => :performance
 
 desc 'Provision systems using ABS (or use already set ABS_RESOURCE_HOSTS) and then execute beaker performance setup and tests'
 rototiller_task :performance do
+  ENV['REF_ARCH'] ||= REF_ARCH_STD
```

Comment:
any reason this default isn't global?

Reply:
This is only needed by 2 out the 45 tasks in the Rakefile, so I am cautious about introducing it as a global.